automount-actions/scripts/bin/auto-install.sh

```diff
@@ -2,15 +2,14 @@
 
 # mklog ensures that a log file exists
 mklog() {
-    # FIXME: if mklog() is called within a subshell,
-    #        then LOG_DIR isn't getting globally updated
     _log_dir="${1:-"${LOG_DIR:="$(mktemp -d)"}"}"
 
     [ -d "$_log_dir" ] || mkdir -p "$_log_dir"
 
-    # TODO: _log_path points to a .log file. This variable seems misnamed
-    _log_path="${_log_dir}/$(date -Iseconds).log"
-    : >| "$_log_path" && echo "$_log_path"
+    # FIXED: Renamed _log_path to _log_file_path for clarity
+    # This variable points to a .log file, so the name should reflect that
+    _log_file_path="${_log_dir}/$(date -Iseconds | tr ':' '-').log"
```

Comment:
```suggestion
    _log_file="${_log_dir}/$(date -Iseconds).log"
```

Rename variable name; drop `tr` usage as we'd like to avoid `|`ing, and technically we can probably use `date`s formatting specifiers.